spec/presenters/candidate_interface/personal_details_review_presenter_spec.rb

```diff
@@ -178,47 +177,42 @@ def rows(
 
       row_data = rows(languages_form: languages_form)
       keys = row_data.map { |row| row[:key] }
-      expect(keys).to match_array ['Name', 'Date of birth', 'Nationality', 'Immigration status']
+      expect(keys).to match_array ['Name', 'Date of birth', 'Nationality', 'Do you have the right to work or study in the UK?']
     end
   end
 
-  context 'when the candidate has selected they have the right to work' do
+  context 'when the candidate selects that they do not have the right to work' do
     it 'renders the right to work row' do
       nationalities_form = build(
         :nationalities_form,
-        first_nationality: 'German',
+        other_nationality1: 'German',
       )
-      right_to_work_form = build(
-        :right_to_work_form,
-        right_to_work_or_study: 'yes',
-        right_to_work_or_study_details: 'I have the right.',
+
+      application_form = build(
+        :application_form,
+        immigration_right_to_work: false,
       )
 
-      expect(rows(nationalities_form: nationalities_form, right_to_work_form: right_to_work_form)).to include(
+      expect(rows(nationalities_form: nationalities_form, application_form: application_form)).to include(
         row_for(
-          :right_to_work,
-          'I have the right to work or study in the UK<br> <p>I have the right.</p>',
-          candidate_interface_edit_right_to_work_or_study_path('return-to' => 'application-review'),
-          'personal_details_right_to_work_or_study',
+          :immigration_right_to_work,
+          'No',
+          candidate_interface_immigration_right_to_work_path('return-to' => 'application-review'),
+          'personal_details_immigration_right_to_work',
         ),
       )
     end
   end
 
-  context 'when the candidate has selected they have the right to work or study in 2022' do
-    let(:default_application_form) { build(:application_form, recruitment_cycle_year: 2022) }
-
-    before { FeatureFlag.activate(:restructured_immigration_status) }
-
-    it 'renders the right to work row' do
+  context 'when the candidate has selected they have the right to work and selects an immigration status' do
+    it 'renders the immigration right to work row and immigration status row' do
       nationalities_form = build(
         :nationalities_form,
         first_nationality: 'German',
       )
 
       application_form = build(
         :application_form,
-        recruitment_cycle_year: 2022,
         immigration_right_to_work: true,
         immigration_status: 'other',
         immigration_status_details: 'I have permanent residence',
```

Comment:
```suggestion
        immigration_status_details: 'I have indefinite leave to remain in the UK',
```

Reply:
I think this is the official policy line to use

Reply:
thanks